src/main/java/net/minecraftforge/network/HandshakeHandler.java

```diff
@@ -329,6 +331,78 @@ void handleConfigSync(final HandshakeMessages.S2CConfigData msg, final Supplier<
         contextSupplier.get().setPacketHandled(true);
         NetworkConstants.handshakeChannel.reply(new HandshakeMessages.C2SAcknowledge(), contextSupplier.get());
     }
+
+    void handleClientReset(HandshakeMessages.S2CReset msg, final Supplier<NetworkEvent.Context> contextSupplier) {
```

Comment:
This method references client-only code, and as such needs to be moved to a client-only class with wrapping.

Specifically, that means this method must remain here, but the implementation has to live in another class, and this method needs to call out to the other class.